syntax = "proto3";

package extraction;

service ExtractionService {
  rpc ExtractDatabase (DatabaseRequest) returns (ExtractResponse);
  rpc ExtractCSV (EmptyRequest) returns (ExtractResponse);
}

message EmptyRequest {}

message DatabaseRequest {
  string table_name = 1;
}

message ExtractResponse {
  bytes data = 1;
}
